content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -182,45 +175,122 @@ Run the following commands on the virtual machine you want to add to the Istio m
     $ sudo apt -y upgrade
     {{< /text >}}
 
+1. Install the root cert to the `/var/run/secrets/istio`
+    
+    {{< text bash >}}
+    $ sudo mkdir -p /var/run/secrets/istio
+    $ sudo cp "${HOME}"/root-cert.pem /var/run/secrets/istio/root-cert.pem
+    {{< /text >}}
+    
+1. Install the token to the `/var/run/secrets/tokens`
+
+    {{< text bash >}}
+    $ sudo  mkdir -p /var/run/secrets/tokens
+    $ sudo cp "${HOME}"/istio-token /var/run/secrets/tokens/istio-token
+    {{< /text >}}
+    
 1. Install the `deb` package containing the Istio virtual machine integration runtime.
 
     {{< text bash >}}
     $ curl -LO https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb
     $ sudo dpkg -i istio-sidecar.deb
     {{< /text >}}
 
-1. Install `root-cert.pem`, `key.pem` and `cert-chain.pem` within the directory `/etc/certs/`.
+1. Install `cluster.env` within the directory `/var/lib/istio/envoy/`.
 
     {{< text bash >}}
-    $ sudo mkdir -p /etc/certs
-    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
+    $ sudo cp "${HOME}"/cluster.env /var/lib/istio/envoy/cluster.env
     {{< /text >}}
 
-1. Install `cluster.env` within `/var/lib/istio/envoy/`.
+1. Install `sidecar.env` within the directory`/var/lib/istio/envoy/`.
 
     {{< text bash >}}
-    $ sudo cp cluster.env /var/lib/istio/envoy
+    $ sudo cp "${HOME}"/sidecar.env /var/lib/istio/envoy/sidecar.env
     {{< /text >}}
 
 1. Add the istiod host to `/etc/hosts`.
 
     {{< text bash >}}
-    $ sudo sh -c 'cat hosts-addendum >> /etc/hosts'
+    $ sudo sh -c 'cat $(eval echo ~$SUDO_USER)/hosts-addendum >> /etc/hosts'
+    {{< /text >}}
+
+1. Install the root certificate in the directory  `/etc/certs`
+
+    {{< text bash >}}
+    $ sudo cp "${HOME}"/root-cert.pem /var/run/secrets/istio/root-cert.pem
     {{< /text >}}
 
 1. Transfer ownership of the files in `/etc/certs/` and `/var/lib/istio/envoy/` to the Istio proxy.
 
     {{< text bash >}}
-    $ sudo chown -R istio-proxy /etc/certs /var/lib/istio/envoy
+    $ sudo mkdir -p /etc/istio/proxy
+    $ sudo chown -R istio-proxy /var/lib/istio /etc/certs /etc/istio/proxy  /var/run/secrets
     {{< /text >}}
 
-1. Start Istio within the virtual machine.
+## Setup policies for virtual machine
```

Comment:
carrier return 238/239.

All section headings require carrier returns on either side in md format.